Cargo.lock

```diff
@@ -3125,12 +3305,25 @@ name = "libsql-sys"
 version = "0.8.0"
 source = "git+https://github.com/tursodatabase/libsql?rev=9b04f1d57d8d098852fed5cd15d6526d89392c4b#9b04f1d57d8d098852fed5cd15d6526d89392c4b"
 dependencies = [
- "bytes 1.8.0",
- "libsql-ffi",
- "libsql-rusqlite",
+ "bytes 1.10.1",
+ "libsql-ffi 0.5.0",
+ "libsql-rusqlite 0.33.0",
+ "once_cell",
+ "tracing",
+ "zerocopy 0.7.35",
+]
+
+[[package]]
+name = "libsql-sys"
+version = "0.9.1"
+source = "git+https://github.com/allan2/libsql?branch=ffi-windows-cp#9364e90ea1ffcd91509607abc00756581d619bda"
+dependencies = [
```

Comment:
issue: libsql got duplicated.

suggestion: Let’s start by updating it on master, via a separate PR.